### PR TITLE
A01: Created a simple expressjs app which prints "Hello World"

### PR DESCRIPTION
**Pull request by shubhmore1212**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import express from "express";
+import greetRoutes from "./routes/public/greet.js";
+
+const PORT = 8000;
```
Comment (mayank-s-dev) [inline index.js]: Use from environment

File: index.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+const app = express();
+
+//ROUTES
```
Comment (mayank-s-dev) [inline index.js]: This shouldn't be in whole caps

File: controller/greet.js [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export const greet = (req, res) => {
+  res.send("Hello World");
```
Comment (mayank-s-dev) [inline controller/greet.js]: Learn about different http errorcode, and add http code in response

Comment (shubhmore1212) [reply controller/greet.js]: Done

File: index.js (hunk 1 above)
Comment (shubhmore1212) [reply index.js]: Done

File: index.js (hunk 2 above)
Comment (shubhmore1212) [reply index.js]: Done

File: index.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import express from "express";
+import dotenv from "dotenv";
+import greetRoutes from "./routes/public/greet.js";
+
+dotenv.config();
+const app = express();
+
+const PORT = process.env.PORT;
```
Comment (mayank-s-dev) [inline index.js]: const PORT = process.env.PORT || 8000

Replace it in such a way.

Comment (shubhmore1212) [reply index.js]: Done

**Pull request merged**